README.md

```diff
@@ -1,2 +1,4 @@
 # xline-operator
-The `xline-operator` is a powerful tool designed to automate the process of bootstrapping, monitoring, snapshotting, and recovering an xline cluster on Kubernetes.  
+
+The xline-operator is a powerful tool designed to automate the process of bootstrapping, monitoring, snapshotting, and
+recovering a xline cluster on Kubernetes.
```

Comment:
It's "an" xline cluster, not a xline cluster.

Reply:
done